### ISSUE TITLE
Exercie 5 content ready for review in project

### ISSUE THREAD

**Issue by yocarrotyo** [yocarrotyo/HammockHelper#4]
Better late than never! :)

**Comment by yocarrotyo** [comment]
wait sorry... hopefully you weren't to quick here -- I forgot to finish the push to remote because it was waiting for my password.

**Comment by pawaitemadisoncollege** [comment]
@yocarrotyo I think these look good. I'm assuming everything passes?

- [ ] Verifying the entities are saved/updated/inserted correctly can be more thorough by implementing and using equals() on each entity - refer to the video Comparing Hibernate Entities with equals in week 5 for more detail.  An example of using that equals method in a unit test would be something like `assertTrue(expectedUser.equals(actualUser));` so that you don't have to compare each instance var/column manually.
- [ ] You'll likely notice, if you haven't already, that writing a Dao for each entity will be very repetitive - check out the generic dao and plan to use it in your project so that you can use a single dao! Note that you will still have unit tests each for each entity(dao) to make sure that annotations and database relationships are set up correctly.

**Comment by yocarrotyo** [comment]
Yep, finally fixed a sql mistake that was causing cleandb to crash last week. I will do the generic DAO this week, definitely don't want to write 2 more of these.
